src/constants.py

```diff
@@ -95,7 +95,7 @@
 ]
 STATIC_EXPANDED_ITEMS_URL = STATIC_SOURCES_URL + "expandedItems.json"
 STATIC_MENUS_URL = STATIC_SOURCES_URL + "hardcodedMenus.json"
-STATIC_EXCEPTIONS_URL = STATIC_SOURCES_URL + "exceptions.json"
+STATIC_ATTRIBUTES_URL = STATIC_SOURCES_URL + "eateryAttributes.json"
```

Comment:
why is it not just `attributes.json`

Reply:
also a good question I think I just saw that the other files had the word eatery in them and wanted to follow suit, but it definitely doesn't need it; I'll change it